1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,5 +1,26 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeDisplay = document.getElementById("timeRemaining");
+  let inputTime = document.getElementById("alarmSet");
+  let seconds = inputTime.value;
```

Comment:
What type is the `seconds` variable here? Object/string/number/Error/boolean?

Reply:
Number

Reply:
Actually, the `value` of an `input` element is always a string as far as I can tell, regardless of the `type` attribute of the element. So you always need to pass it through `parseInt(input, 10)` and check for `NaN`